tests/test_routes.py

```diff
@@ -68,6 +68,21 @@ def _create_orders(self, count):
             orders.append(test_order)
         return orders
 
+    def _create_orders_items(self, count):
```

Comment:
Dont think we need this function. We can modify _create_orders() as - 

    def _create_orders(self, count):
        """Factory method to create orders in bulk"""
        orders = []
        for _ in range(count):
            item = OrderItemFactory()
            test_order = OrderFactory(order_items = [item])

            resp = self.app.post(
                BASE_URL, json=test_order.serialize(), content_type=CONTENT_TYPE_JSON
            )
            self.assertEqual(
                resp.status_code, status.HTTP_201_CREATED, "Could not create test order"
            )
            new_order = resp.get_json()
            test_order.id = new_order["id"]
            assert len(test_order.order_items) == 1
            order_items = new_order["order_items"]
            for i, item in enumerate(order_items):
                test_order.order_items[i].item_id = item["item_id"]
            orders.append(test_order)
        print(f'orders :: {orders}, count :: {count}')
        for order in orders:
            print(order.order_items)
        return orders

Reply:
Updated the _create_order method

Reply:
We can delete this function